### PR TITLE
[Feature] [DAF-4278]

### PR DESCRIPTION
**Pull request by ghost**
I am currently working on ticket [DAF-4278](https://dw-ml-nfc.atlassian.net/browse/DAF-4278), which requires pausing the player when the user taps on the `X` button (on the left side) of the PIP mode. However, there is currently no callback for this action. Therefore, I have added a parameter called `isRestorePIPButtonPressed` to the `exitingPIP` event. This parameter is set to `true` when the user presses the `restore` button (on the right side of the PIP).

[DAF-4278]: https://dw-ml-nfc.atlassian.net/browse/DAF-4278?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by mvn-thanhluu-hn — APPROVED** [review]
LGTM.

**Review by noboru-i — APPROVED** [review]
LGTM.

@HiroshiOshiro 
Could you review this PR? 🙏

File: ios/Classes/BetterPlayer.h [hunk 1]
```diff
@@ -38,6 +38,7 @@ NS_ASSUME_NONNULL_BEGIN
 @property(nonatomic) float playerRate;
 @property(nonatomic) int overriddenDuration;
 @property(nonatomic) AVPlayerTimeControlStatus lastAvPlayerTimeControlStatus;
+@property(nonatomic) bool isRestorePIPButtonPressed;
```
Comment (HiroshiOshiro) [inline ios/Classes/BetterPlayer.h]: `isRestore` sounds like `re-start` PIP window for me. 
Maybe just `close` or `dismiss` would be suitable in this case.

**Review by HiroshiOshiro — CHANGES_REQUESTED** [review]
Please consider the naming again.

File: ios/Classes/BetterPlayer.h (hunk 1 above)
Comment (ghost) [reply ios/Classes/BetterPlayer.h]: @HiroshiOshiro 
sorry, as I mentioned before 
> This parameter is set to true when the user presses the restore button (on the right side of the PIP).

I think `close` or `dismiss` is more suitable for X button ( on the left side ). Maybe I should change to `isRestoreButtonPIPPressed` or do you have any other recommendations?

Comment (HiroshiOshiro) [reply ios/Classes/BetterPlayer.h]: I see, sorry I misunderstood that.

In that case, it is going back  to our app. If I, I would call the right side button as `backToAppButton` so I think `isBackToAppButtonPressed` would be suitable.

Comment (ghost) [reply ios/Classes/BetterPlayer.h]: @HiroshiOshiro sorry for late, I changed the name, pls check it 🙇

**Review by HiroshiOshiro — APPROVED** [review]
LGTM, Thanks!

**Pull request merged**